src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPool.scala

```diff
@@ -30,7 +31,7 @@ class ErgoMemPool private[mempool](pool: OrderedTxPool)(implicit settings: ErgoS
 
   override def getAll(ids: Seq[ModifierId]): Seq[ErgoTransaction] = ids.flatMap(pool.get)
 
-  override def getAllPrioritized: Seq[ErgoTransaction] = pool.orderedTransactions.values.toSeq.reverse
+  override def getAllPrioritized: Seq[ErgoTransaction] = pool.orderedTransactions.values.toSeq
```

Comment:
So it was a bug? Please add ScalaDocs to avoid confusion in future.

Reply:
Not bug, but sorting now done in descending order, so need for reverse

Reply:
Well, why not make it clear with ScalaDoc?

Reply:
Comment was done for a method definition in the base class, copied scaladoc to the concrete definition also